src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -120,6 +120,18 @@
         <type>Number</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>RecurringDonationNaming__c</fullName>
```

Comment:
@andrewyu-salesforce Since there is already a checkbox that indicates the "state" of RD naming, I think this field should be something like `RecurringDonationNamingFormat__c`; possibly with a label such as "Recurring Donation Name Format".
cc @sam-knox

Reply:
I think both work. I include 2 screen shots on what it will look at in the UI. @sam-knox 
<img width="1164" alt="Screen Shot 2020-06-30 at 12 53 03 PM" src="https://user-images.githubusercontent.com/57271120/86170885-ab697300-bad0-11ea-8b79-dab5cf931dcd.png">
<img width="1136" alt="Screen Shot 2020-06-30 at 12 52 52 PM" src="https://user-images.githubusercontent.com/57271120/86171014-e2d81f80-bad0-11ea-94d8-4fbeb3565c43.png">

Reply:
We're just talking about the label and help text for the custom setting here right? For the field label in NPSP Settings, I do like "Recurring Donation Name Format" better. The label used for Opportunity names is "Opportunity Name Format".